specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2462,14 +2512,22 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+			      "UserAssigned"
           ],
           "x-ms-enum": {
             "name": "IdentityType",
             "modelAsString": false
           }
-        }
-      },
+        },
+		    "userAssignedIdentities":{
+		      "type": "object",
+		      "description": "Dictionary to map userAssignedIdentityID strings to userAssignedIdentity properties",
```

Comment:
As it's a dictionary, can there be multiple identities set to same account?
If so, please update sample to include multiple identities, and please clarify how server find the identity to be used in encryption.
If so, we might should not use dictionary here.

Reply:
If only one allowed, could we at least add the information to the description?

Reply:
MultipleIdentities are allowed in the top level "userAssignedIdentities" dictionary.

Reply:
@anthonykjose  Can you please weigh in here ?